docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -7,7 +7,7 @@
     "tags": []
```

Comment:
<p>Space between "Above-Ground Biomass" and "(AGB)" in your first sentence under "About the Dataset" (thank you for adding that acronym in!)</p><p> </p><p>If you wanted to be more consistent with other notebooks, I'd put the disclaimer under the "Run this Notebook" section, but it's definitely good that it's there!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thank you for the great eyes! updated the notebook accordingly. :)</p>